bee-crypto/Cargo.toml

```
@@ -11,7 +11,7 @@ keywords = ["iota", "tangle", "bee", "framework", "crypto"]
 homepage = "https://www.iota.org"
 
 [dependencies]
-bee-ternary = { path = "../bee-ternary" }
+bee-ternary = "0.3.1-alpha"
```

You can do this so it can use the latest bee-ternary which hasn't published to crates.io
```suggestion
bee-ternary = { version = ""0.3.1-alpha"", path = "../bee-ternary" }
```